commit 2fcd8e84975f860f2011016e3718b5aa1449aa62 (refs/changes/58/258/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-08-09T20:46:07+00:00 (3 years, 2 months ago)
    
    joyent/node-cueball#10 want support for 0.10
    Reviewed by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 042c0ed..f7d9ccc 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "0.3.12",
+  "version": "0.3.13",
   "description": "",
   "main": "lib/index.js",
   "dependencies": {
@@ -10,7 +10,7 @@
     "extsprintf": ">=1.3.0 <2.0.0",
     "ipaddr.js": ">=1.1.0 <2.0.0",
     "mooremachine": ">=1.2.0 <2.0.0",
-    "named-client": "git://github.com/arekinath/node-named-client#v0.3.4",
+    "named-client": "git://github.com/arekinath/node-named-client#v0.3.5",
     "node-uuid": ">=1.4.7 <2.0.0",
     "posix-getopt": ">=1.2.0 <2.0.0",
     "restify-clients": ">=1.1.2 <2.0.0",
